CRM/Case/ActionMapping.php

```diff
@@ -126,7 +137,40 @@ public function createQuery($schedule, $phase, $defaultParams) {
       throw new CRM_Core_Exception("Invalid date field");
     }
 
-    $query['casDateField'] = $schedule->start_action_date;
+    if ($schedule->start_action_date == 'case_status') {
```

Comment:
Don't think this looks right, for a date field to be returning the case status. Can you explain why it is like this? IT is a PR going to core and if it's not proper, we might not be able to get this in.

Reply:
This is not the case_status that is used here. I have changed the name 'case_status' to 'case_status_change_date' to be a little better.
This might look a little hacky because we dont have a better way to get when a case status has changed in civicrm core currently. 
Pls also see this comment https://github.com/compucorp/civicrm-core/pull/31#discussion_r327248512